File: hazi2/feladat.py
# Nev: Kurucz László
# Neptun: Z5RFY1
# h: h373677


def nyertes_korok(adam, ellenfel):
    if len(adam) != len(ellenfel) or len(adam) == 0 and len(ellenfel) == 0: return -1

    adam_nyert = 0

    for i in range(len(adam)):
        if(adam[i] > ellenfel[i]): adam_nyert += 1

    return adam_nyert

# print(nyertes_korok([], []))

File: hazi7/feladat.js
// Nev: Kurucz László
// Neptun: Z5RFY1
// h: h373677

function matek(param) {
	if (param == undefined || param == null) return 0;
	else if (typeof param == "string") return 1;
	else if (Number.isInteger(param))
		return param % 2 == 0 ? Math.pow(param, param) : Math.pow(param - 1, 2);
	else return null;
}

// console.log(matek(8));
